src/modules/tests/postComments.test.js

```diff
@@ -0,0 +1,17 @@
+import { postComments } from '../comments.js';
+
+test('should notify a comment has been posted', async () => {
+  const data = {
+    item_id: '52819',
+    username: 'Junaid',
+    comment: 'Hello',
+  };
+
+  const response = await postComments(
+    data.item_id,
+    data.username,
+    data.comment,
+  );
+
+  expect(response).toBe('Created');
```

Comment:
- [ ] Great job fixing the other changes, but you still have one failing test so make sure that you fix it and ensure that the app is working perfectly
<img width="605" alt="image" src="https://github.com/iamsjunaid/js-capstone-2/assets/37901469/7e635181-4a75-4c08-973d-4bda83cae722">

Reply:
The app is perfectly fine on our computers and all the tests cases are passing perfectly fine. I am attaching a screenshot of test cases done on my computer. Please go through it once again.

![Screenshot (270)](https://github.com/iamsjunaid/js-capstone-2/assets/61676854/b0ff162d-ee22-46e6-a5b0-aacc4b4b2597)
![Screenshot (269)](https://github.com/iamsjunaid/js-capstone-2/assets/61676854/5509cf6c-ac23-4829-a06b-e9875b8c8e7c)